### ISSUE TITLE
BUG: Beekeeper studio (DEB) won't start in WSL2

### ISSUE THREAD

**Issue by ThaDaVos** [beekeeper-studio/beekeeper-studio#364]
**Details:**
Details about your question

How does one start the application from command line?

**Version Information [might be relevant to your issue]**
Operating System: Ubuntu -> WSL 2 (using an x-server)
App Version [help -> about]: Latest (cannot check further as I cannot start the application)

**Comment by ThaDaVos** [comment]
I installed it through deb yes and that doesn't work

**Comment by caramalac** [comment]
I have the same problem. WLS2 running Ubuntu 20.04, and don´t get uninstall.

**Comment by rathboma** [comment]
This was partially developed on WSL2. My guess is that some system packages are missing. Can you try installing these packages?

`sudo apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev`

**Comment by rathboma** [comment]
@caramalac @ThaDaVos did the instructions above help?

**Comment by ntr-808** [comment]
I'm assuming I'm having the same issue as the above posters which is that after installing via the deb there is no `beekeeper-studio` in my path to actually start the app.

**Comment by ThaDaVos** [comment]
Sorry for being quiet busy work days,

I fixed it by running the following command after install based on @ntr-808 answer:
```bash
sudo ln -s /opt/Beekeeper\ Studio/beekeeper-studio /usr/bin/beekeeper-studio
```

**Comment by ThaDaVos** [comment]
I have on other issue though, after it's launched (I am using GWSL as x-server in windows) - I cannot move the window, only enlarge it by double clicking but after that I cannot minimize it etc...

![image](https://user-images.githubusercontent.com/5251028/115354207-a8538680-a1b9-11eb-8900-c74edef28f37.png)

**Comment by ThaDaVos** [comment]
Seems to be only an issue with GWSL - ignore that, it works fine with X410

**Comment by rathboma** [comment]
Great! Going to close this issue. 

Created [another ticket](https://github.com/beekeeper-studio/beekeeper-studio/issues/634) to explicitly add beekeeper-studio to /usr/bin

**Comment by laporeon** [comment]
I'm trying to use Beekeeper with WSL2 as well, but so far I'm not doing things right, I guess. Distro and version i'm using is Ubuntu 18.04.

Doing some researches, i found this topic so I installed Beekeeper on WSL using the official tutorial: https://docs.beekeeperstudio.io/installation/#linux-installation

But when i type "beekeeper-studio", that's the response I got:

![beekeeper](https://user-images.githubusercontent.com/34722707/144724851-71c3fc54-beb5-47de-9814-26b87dbc5367.png)

Is there a way to use beekeeper with WSL2 without using Docker? 

What am I doing wrong?
